app/code/Magento/NewRelicReporting/etc/csp_whitelist.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<csp_whitelist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Csp:etc/csp_whitelist.xsd">
+    <policies>
+        <policy id="script-src">
+            <values>
+              <value id="newrelic" type="host">*.newrelic.com</value>
+              <value id="newrelic-data" type="host">*.nr-data.com</value>
```

Comment:
while testing, I discovered that this pattern must be changed to `*.nr-data.net` (see `connect-src`)

Reply:
Resolved on the original branch. Once re-opened should be updated with the change